### PR TITLE
Add support for toml files

### PR DESCRIPTION
**Pull request by eganjs**
Hi, I've put together a small PR for adding support for toml files!

Description of changes:
* Add TomlFile
* Add @iarna/toml dependency to enable writing toml files
* Fix bug where bundledDeps in NodeProject would parse scoped packages incorrectly

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/toml.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { FileBase, FileBaseOptions, IResolver } from './file';
```
Comment (eladb) [inline src/toml.ts:1]: Please add api docstrings to all public APIs

**Review by eladb — COMMENTED** [review]
Looks great. Minor comments

**Comment by eladb** [comment]
Please run `yarn build` and commit the updated snapshots

File: src/toml.ts (hunk 1 above)
Comment (eganjs) [reply src/toml.ts:2]: I've added them for `TomlFile` and `obj` in `TomlFileOptions`, let me know if there's anything I have missed!

**Comment by eganjs** [comment]
> Please run `yarn build` and commit the updated snapshots

I've run the command and committed the updated API.md, is this the snapshot you were referring to?

**Review by eladb — APPROVED** [review]

**Pull request merged**